### PR TITLE
US122137/add skeleton to right panel.

### PR DESCRIPTION
**Pull request by mbosiljevac**
image

### CODE REVIEW

File: components/right-panel/consistent-evaluation-feedback-presentational.js [hunk 1]
```diff
@@ -195,7 +238,9 @@ class ConsistentEvaluationFeedbackPresentational extends LocalizeConsistentEvalu
 				: null;
 			this._setFeedbackSummaryInfo();
 			return html`
+				${this._renderSkeleton()}
 				<d2l-consistent-evaluation-right-panel-block
+					class="d2l-consistent-evaluation-feedback-block"
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-feedback-presentational.js:243]: I think we want `aria-hidden="${this.skeleton}"` here as well

File: components/right-panel/consistent-evaluation-outcomes.js [hunk 2]
```diff
@@ -26,12 +27,65 @@ class ConsistentEvaluationOutcomes extends LocalizeConsistentEvaluation(LitEleme
...
+				<div class="d2l-skeleton-outcomes-footer d2l-skeletize"></div>
+			</div>
+		</div>`;
 	}
 
 	render() {
 		return html`
+			${this._renderSkeleton()}
 			<d2l-consistent-evaluation-right-panel-block
+				class="d2l-consistent-evaluation-outcomes-block"
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-outcomes.js:88]: I think we want `aria-hidden="${this.skeleton}"` here as well

File: components/right-panel/consistent-evaluation-rubric.js [hunk 3]
```diff
@@ -329,7 +385,9 @@ class ConsistentEvaluationRubric extends LocalizeConsistentEvaluation(RtlMixin(L
 
 	render() {
 		return html`
+			${!this.isPopout ? this._renderSkeleton() : ''}
 			<d2l-consistent-evaluation-right-panel-block
+				class="d2l-consistent-evaluation-rubric-block"
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-rubric.js:390]: I think we want `aria-hidden="${this.skeleton}"` here as well

**Review by jeroach — COMMENTED** [review]
vdiff tests look good - so does the full render

Just a few missing attributes, and this is good to go :)

**Review by jeroach — APPROVED** [review]

**Review by JamieTran — APPROVED** [review]
Nice work! I wonder if we want loading skeletons for the mobile view as well... That shouldn't be encompassed in this PR though :)

**Comment by mbosiljevac** [comment]
> Nice work! I wonder if we want loading skeletons for the mobile view as well... That shouldn't be encompassed in this PR though :)

Ye for sure! That's something we should probably do

**Pull request merged**